ninja.lua

```diff
@@ -348,7 +363,11 @@ local function compilation_rules(cfg, toolset, pch)
 	local cxx = toolset.gettoolname(cfg, "cxx")
 	local ar = toolset.gettoolname(cfg, "ar")
 	local link = toolset.gettoolname(cfg, iif(cfg.language == "C", "cc", "cxx"))
-	local rc = toolset.gettoolname(cfg, "rc")
+	-- only compile rc files on windows
```

Comment:
I checked premake-core:
- They don't seems to filter out resource files for non-windows. (gmake/gmake2/codelite)

How about cross compilation?

Alternative for user is

```lua
filter {'system:windows'}
    files { 'my_resource.rc' }
filter {}
```